CHANGES.rst

```diff
@@ -8,6 +8,106 @@ https://hyperspy.readthedocs.io/en/latest/user_guide/changes.html
 
 .. towncrier release notes start
 
+Hyperspy 1.7.0 (2022-04-26)
+===========================
+
+New features
+------------
+
+- Add ``filter_zero_loss_peak`` argument to the :py:meth:`~._signals.eels.EELSSpectrum.spikes_removal_tool` method (`#1412 <https://github.com/hyperspy/hyperspy/issues/1412>`_)
```

Comment:
the link to the method does not work

Reply:
I noticed too, but I don't know the reason... :(

Reply:
Searching the API-docs, only [_signals.signal1d.Signal1D.spikes_removal_tool](https://hyperspy.org/hyperspy-doc/current/api/hyperspy._signals.signal1d.html?highlight=spikes_removal_tool#hyperspy._signals.signal1d.Signal1D.spikes_removal_tool) exists, not as method of the `eels` class.

Reply:
Looking at previous version, it seems that it stopped working in version 1.6. I will leave for another PR.